### ISSUE TITLE
Django polish: Wiki header JS

### ISSUE THREAD

**Issue by gdude2002** [python-discord/site#211]
Once #202 is merged and #200 is solved, consider writing a piece of JavaScript that transforms the wiki headers from fully-linked titles, to titles with a linked paragraph mark that shows up on hover - basically how the old site worked.

**Comment by Den4200** [comment]
Dewikification makes this no longer relevant.
